crates/rooch-store/src/transaction_store/mod.rs

```diff
@@ -73,8 +74,21 @@ impl TransactionDBStore {
         self.typed_tx_store.kv_get(hash)
     }
 
-    //TODO implements get type tx by index
-    pub fn get_tx_by_index(&self, _cursor: u64, _limit: u64) -> Result<Vec<TypedTransaction>> {
+    pub fn get_tx_by_hash_and_index(&self, hash: H256, _index: u64) -> Result<TypedTransaction> {
```

Comment:
In rooch, the index of tx is determinate, we do not need `get_tx_by_hash_and_index`.

Reply:
Okay. Will use https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gettransactionbyhash.